package.json

```diff
@@ -37,6 +37,7 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "@dvsa/cvs-type-definitions": "^3.3.1",
```

Comment:
This should be updated to include the latest enum change from Leo's pr

Reply:
Should these types be in dependencies rather than devDependencies?

Reply:
I can move it there if everyone wants it to be there 👍🏼

Reply:
That is where Kris has put it on his PR, also where it is on v3 tech records.

Reply:
in v3, it's in prod deps

Reply:
Yes exactly, that's what I was saying. In Kris's PR and v3 tech records it's in dependencies not devDependencies.